roles/addons/nhc/tasks/main.yml

```diff
@@ -0,0 +1,54 @@
+---
+
+- name: include_vars ░ Gather OS specific variables
+  # This task gathers variables defined in OS specific files.
+  #
+  # Search vars in:
+  #  - <distribution>_<major>.yml    # eg. CentOS_8.yml
+  #  - <os_family>_<major>.yml       # eg. RedHat_8.yml
+  #  - <distribution>.yml            # eg. CentOS.yml
+  #  - <os_family>.yml               # eg. RedHat.yml
+  #
+  # If no OS specific file is found, the role will default to vars/main.yml
+  #
+  include_vars: "{{ item }}"
+  with_first_found:
+    - files:
+        - "vars/{{ ansible_facts.distribution | replace(' ','_') }}_{{ ansible_facts.distribution_major_version }}.yml"
+        - "vars/{{ ansible_facts.os_family }}_{{ ansible_facts.distribution_major_version }}.yml"
+        - "vars/{{ ansible_facts.distribution | replace(' ','_') }}.yml"
+        - "vars/{{ ansible_facts.os_family }}.yml"
+      skip: true
+  tags:
+    - internal
+    - package
+
+- name: "package █ Install {{ nhc_packages_to_install | join(' ') }}"
+  package:
+    name: "{{ nhc_packages_to_install }}"
+    state: present
+
+- name: package █ Install optional custom packages
```

Comment:
It should be possible to combine the two lists `nhc_packages_to_install` and `nhc_custom_packages_to_install` in a single task.

Reply:
Yes, this would speedup a bit execution. I will do that.